src/controllers/router-store/test.tsx

```diff
@@ -567,7 +567,7 @@ describe('RouterStore', () => {
           expect.objectContaining({ route }),
           undefined
         );
-        expect(wrapper.html()).toEqual('home');
```

Comment:
I wonder why was html==='home'? I thought `component: () => null,` should render `null`

Reply:
Not sure about the old tests but in the new one I am rendering something with the text home in it to make sure things are correctly rendered.